server/logic/tournament_status_logic.js

```diff
@@ -88,7 +90,9 @@ function getNextMatchForTable (tableId, amountOfMatches = 1) {
 function publishMatchAvailable () {
   getCurrentMatch().then(match => {
     if (match) {
-      publishUpdateMsg('CurrentMatch', convertMatchTimeToToday(match))
+      offsetMatch(match).then(newMatch => {
```

Comment:
Here you run `offsetMatch` first, and then `convertMatchTimeToToday`.
In line 106 it's the opposite. Is there a reason for that of does they always come the same and should be encapsulated together?